Plugins/Public/conn_plugin/Main.cpp

```diff
@@ -254,11 +227,7 @@ void MoveClient(unsigned int client, unsigned int targetBase)
 			return;
 		}
 		PrintUserCmdText(client, L"Player base renamed/destroyed, ship redirected to a proxy base");
-		pub::Player::ForceLand(client, proxyBaseID); // beam
-		if (returnSys != system)
-		{
-			SimulateF1(client, proxyBaseID);
-		}
+		HkBeamById(client, targetBase);
```

Comment:
We know this can't work because in this case Universe::get_base(targetBase) failed

```suggestion
		HkBeamById(client, proxyBaseID);
```